war/src/main/js/config-scrollspy.less

```diff
@@ -11,6 +11,7 @@
  * Widget styles
  */
 @import "widgets/config/tabbar";
+@import "widgets/add/addform";
```

Comment:
@gusreiber your attempts to "push [stuff] through in a small window of time" is actually slowing you (and this PR) down because others then have to get involved to clean things up.